From df53d2e76ffd1105c5d2a4cb22cac576b80e2afd Mon Sep 17 00:00:00 2001
From: Barry Wardell <barry.wardell@gmail.com>
Date: Tue, 13 Dec 2011 11:07:54 +0000
Subject: [PATCH 2/3] Update FFTW options for all relevant machines.

* The FFTW_DIR option has been renamed to FFTW3_DIR for machines which actually provide version 3.
* Remove the unused FFTW_LIBS option for machines which provide version 3.
---
 mdb/optionlists/abe-mvapich2.cfg      |    3 +--
 mdb/optionlists/athena-intel.cfg      |    2 +-
 mdb/optionlists/bethe.cfg             |    2 +-
 mdb/optionlists/frost.cfg             |    2 +-
 mdb/optionlists/hlrb2.cfg             |    2 +-
 mdb/optionlists/hopper-intel.cfg      |    2 +-
 mdb/optionlists/kraken-gcc.cfg        |    2 +-
 mdb/optionlists/kraken-intel.cfg      |    2 +-
 mdb/optionlists/kraken-pgi10.cfg      |    2 +-
 mdb/optionlists/kraken.cfg            |    2 +-
 mdb/optionlists/lincoln-mvapich2.cfg  |    3 +--
 mdb/optionlists/philip-mpich.cfg      |    2 +-
 mdb/optionlists/queenbee-mvapich.cfg  |    2 +-
 mdb/optionlists/queenbee-mvapich2.cfg |    2 +-
 mdb/optionlists/supermuc.cfg          |    2 +-
 mdb/optionlists/tezpur-mvapich2.cfg   |    2 +-
 16 files changed, 16 insertions(+), 18 deletions(-)

diff --git a/mdb/optionlists/abe-mvapich2.cfg b/mdb/optionlists/abe-mvapich2.cfg
index d1e6efc..b17c96f 100644
--- a/mdb/optionlists/abe-mvapich2.cfg
+++ b/mdb/optionlists/abe-mvapich2.cfg
@@ -83,8 +83,7 @@ F90_WARN_FLAGS =
 BLAS_DIR  = /usr/local/intel/mkl/10.2.1.017/lib/em64t
 BLAS_LIBS =  -Wl,--start-group mkl_intel_lp64 mkl_intel_thread mkl_core -Wl,--end-group   iomp5   pthread
 
-FFTW_DIR  = /usr/apps/math/fftw/fftw-3.1.2/intel10
-FFTW_LIBS = fftw3
+FFTW3_DIR  = /usr/apps/math/fftw/fftw-3.1.2/intel10
 
 LAPACK_DIR  = /usr/local/intel/mkl/10.2.1.017/lib/em64t
 LAPACK_LIBS = -Wl,--start-group mkl_lapack mkl_intel_lp64 mkl_intel_thread mkl_core -Wl,--end-group
diff --git a/mdb/optionlists/athena-intel.cfg b/mdb/optionlists/athena-intel.cfg
index 987bc1c..3ebc25d 100644
--- a/mdb/optionlists/athena-intel.cfg
+++ b/mdb/optionlists/athena-intel.cfg
@@ -85,7 +85,7 @@ F77_WARN_FLAGS =
 BLAS_DIR  = /opt/intel/Compiler/11.1/038/mkl/lib/em64t
 BLAS_LIBS = -Wl,--start-group mkl_intel_lp64 mkl_intel_thread mkl_core -Wl,--end-group   iomp5   pthread
 
-FFTW_DIR = /opt/fftw/3.2.2.1
+FFTW3_DIR = /opt/fftw/3.2.2.1
 
 GSL_DIR = /sw/xt/gsl/1.14/cnl2.2_intel11.1.038
 
diff --git a/mdb/optionlists/bethe.cfg b/mdb/optionlists/bethe.cfg
index 03c4492..4169ca6 100644
--- a/mdb/optionlists/bethe.cfg
+++ b/mdb/optionlists/bethe.cfg
@@ -86,7 +86,7 @@ F90_WARN_FLAGS =
 BLAS_DIR  = /usr/local/apps/intel/intel_cc_11_1/mkl/lib/em64t
 BLAS_LIBS = mkl_intel_lp64 mkl_blas95_lp64 mkl_core guide mkl_sequential pthread ifport
 
-#FFTW_DIR = /usr/local/packages/fftw-3.1.2-intel10.1
+#FFTW3_DIR = /usr/local/packages/fftw-3.1.2-intel10.1
 
 GSL_DIR = /usr/local/apps/gsl-1.13-intel-intel_11.1
 
diff --git a/mdb/optionlists/frost.cfg b/mdb/optionlists/frost.cfg
index 218d7bf..e0b4514 100644
--- a/mdb/optionlists/frost.cfg
+++ b/mdb/optionlists/frost.cfg
@@ -79,7 +79,7 @@ F77_WARN_FLAGS =
 BLAS_DIR  = /contrib/bgl/lib
 BLAS_LIBS = blas440
 
-FFTW_DIR = /contrib/bgl/fftw-3.2-double
+FFTW3_DIR = /contrib/bgl/fftw-3.2-double
 
 GSL_DIR = /home/diener/gsl-1.14
 
diff --git a/mdb/optionlists/hlrb2.cfg b/mdb/optionlists/hlrb2.cfg
index fa054b6..2488925 100644
--- a/mdb/optionlists/hlrb2.cfg
+++ b/mdb/optionlists/hlrb2.cfg
@@ -83,7 +83,7 @@ F90_WARN_FLAGS = -warn all
 BLAS_DIR  = /lrz/sys/intel/mkl91_06/lib/64
 BLAS_LIBS = mkl_lapack mkl_ipf guide pthread
 
-FFTW_DIR = /lrz/sys/fftw/3.1.1
+FFTW3_DIR = /lrz/sys/fftw/3.1.1
 
 GSL_DIR = /lrz/sys/libraries/gsl/1.12
 
diff --git a/mdb/optionlists/hopper-intel.cfg b/mdb/optionlists/hopper-intel.cfg
index 5b02f3d..6efbf05 100644
--- a/mdb/optionlists/hopper-intel.cfg
+++ b/mdb/optionlists/hopper-intel.cfg
@@ -72,7 +72,7 @@ F77_WARN_FLAGS =
 BLAS_DIR  = NO_BUILD
 BLAS_LIBS = -mkl
 
-FFTW_DIR = /opt/fftw/3.2.2.1
+FFTW3_DIR = /opt/fftw/3.2.2.1
 
 GSL_DIR = /usr/common/usg/gsl/1.14
 
diff --git a/mdb/optionlists/kraken-gcc.cfg b/mdb/optionlists/kraken-gcc.cfg
index fa63559..c060b7f 100644
--- a/mdb/optionlists/kraken-gcc.cfg
+++ b/mdb/optionlists/kraken-gcc.cfg
@@ -88,7 +88,7 @@ F90_WARN_FLAGS = -Wall -Wshadow -Wsurprising
 #BLAS_LIBS = blas
 BLAS_DIR = BUILD
 
-FFTW_DIR = /sw/xt/fftw/3.1.2/cnl2.2_gnu4.4.3
+FFTW3_DIR = /sw/xt/fftw/3.1.2/cnl2.2_gnu4.4.3
 
 # libgsl.a seems incompatible; it was probably built with an older
 # version of gcc
diff --git a/mdb/optionlists/kraken-intel.cfg b/mdb/optionlists/kraken-intel.cfg
index 3615ea1..424f0d8 100644
--- a/mdb/optionlists/kraken-intel.cfg
+++ b/mdb/optionlists/kraken-intel.cfg
@@ -85,7 +85,7 @@ F77_WARN_FLAGS =
 BLAS_DIR  = /opt/intel/Compiler/11.1/038/mkl/lib/em64t
 BLAS_LIBS = -Wl,--start-group mkl_intel_lp64 mkl_intel_thread mkl_core -Wl,--end-group   iomp5   pthread
 
-FFTW_DIR = /opt/fftw/3.2.2.1
+FFTW3_DIR = /opt/fftw/3.2.2.1
 
 GSL_DIR = /sw/xt/gsl/1.14/cnl2.2_intel11.1.038
 
diff --git a/mdb/optionlists/kraken-pgi10.cfg b/mdb/optionlists/kraken-pgi10.cfg
index 2848378..52bbf97 100644
--- a/mdb/optionlists/kraken-pgi10.cfg
+++ b/mdb/optionlists/kraken-pgi10.cfg
@@ -83,7 +83,7 @@ F77_WARN_FLAGS =
 BLAS_DIR  = /sw/xt5/blas/ref/cnl2.1_pgi7.2.3/lib
 BLAS_LIBS = blas
 
-FFTW_DIR = /sw/xt5/fftw/3.2.1/cnl2.1_pgi7.2.5
+FFTW3_DIR = /sw/xt5/fftw/3.2.1/cnl2.1_pgi7.2.5
 
 GSL_DIR = /sw/xt5/gsl/1.11/cnl2.1_pgi7.2.3
 
diff --git a/mdb/optionlists/kraken.cfg b/mdb/optionlists/kraken.cfg
index 9441cf1..0574fae 100644
--- a/mdb/optionlists/kraken.cfg
+++ b/mdb/optionlists/kraken.cfg
@@ -79,7 +79,7 @@ BLAS_DIR  = /sw/xt/blas/ref/cnl2.2_pgi9.0.1/lib
 
 BLAS_LIBS = blas
 
-FFTW_DIR = /sw/xt/fftw/3.1.2/cnl2.2_pgi9.0.1
+FFTW3_DIR = /sw/xt/fftw/3.1.2/cnl2.2_pgi9.0.1
 
 GSL_DIR = /sw/xt/gsl/1.14/cnl2.2_pgi10.4.0
 
diff --git a/mdb/optionlists/lincoln-mvapich2.cfg b/mdb/optionlists/lincoln-mvapich2.cfg
index c713098..3b1f410 100644
--- a/mdb/optionlists/lincoln-mvapich2.cfg
+++ b/mdb/optionlists/lincoln-mvapich2.cfg
@@ -80,8 +80,7 @@ F90_WARN_FLAGS =
 BLAS_DIR  = /usr/local/intel/mkl/10.2.1.017/lib/em64t
 BLAS_LIBS =  -Wl,--start-group mkl_intel_lp64 mkl_intel_thread mkl_core -Wl,--end-group   iomp5   pthread
 
-FFTW_DIR  = /usr/apps/math/fftw/fftw-3.1.2/intel10
-FFTW_LIBS = fftw3
+FFTW3_DIR  = /usr/apps/math/fftw/fftw-3.1.2/intel10
 
 LAPACK_DIR  = /usr/local/intel/mkl/10.2.1.017/lib/em64t
 LAPACK_LIBS = -Wl,--start-group mkl_lapack mkl_intel_lp64 mkl_intel_thread mkl_core -Wl,--end-group
diff --git a/mdb/optionlists/philip-mpich.cfg b/mdb/optionlists/philip-mpich.cfg
index 2966832..5c78ae9 100644
--- a/mdb/optionlists/philip-mpich.cfg
+++ b/mdb/optionlists/philip-mpich.cfg
@@ -82,7 +82,7 @@ F90_WARN_FLAGS =
 BLAS_DIR  = /usr/local/compilers/Intel/mkl-10.1.0.015/lib/em64t
 BLAS_LIBS = -Bstatic mkl_em64t guide -Bdynamic pthread
 
-FFTW_DIR = /usr/local/packages/fftw/3.2/intel-11.1-mpich-1.2.7p1
+FFTW3_DIR = /usr/local/packages/fftw/3.2/intel-11.1-mpich-1.2.7p1
 
 GSL_DIR = /usr/local/packages/gsl/1.9/intel-11.1-mpich-1.2.7p1
 
diff --git a/mdb/optionlists/queenbee-mvapich.cfg b/mdb/optionlists/queenbee-mvapich.cfg
index dd9107a..e976a99 100644
--- a/mdb/optionlists/queenbee-mvapich.cfg
+++ b/mdb/optionlists/queenbee-mvapich.cfg
@@ -84,7 +84,7 @@ F90_WARN_FLAGS =
 BLAS_DIR  = /usr/local/compilers/Intel/mkl-10.2/lib/em64t
 BLAS_LIBS = mkl_solver_lp64 mkl_intel_lp64 mkl_intel_thread mkl_core   iomp5 guide   pthread
 
-FFTW_DIR = /usr/local/packages/fftw/3.2/intel-11.1-mvapich-1.1
+FFTW3_DIR = /usr/local/packages/fftw/3.2/intel-11.1-mvapich-1.1
 
 GSL_DIR = /usr/local/packages/gsl/1.9/intel-11.1-mvapich-1.1
 
diff --git a/mdb/optionlists/queenbee-mvapich2.cfg b/mdb/optionlists/queenbee-mvapich2.cfg
index 3687dc6..9204db9 100644
--- a/mdb/optionlists/queenbee-mvapich2.cfg
+++ b/mdb/optionlists/queenbee-mvapich2.cfg
@@ -92,7 +92,7 @@ F90_WARN_FLAGS =
 BLAS_DIR  = /usr/local/compilers/Intel/mkl-10.2/lib/em64t
 BLAS_LIBS = mkl_solver_lp64 mkl_intel_lp64 mkl_intel_thread mkl_core   iomp5 guide   pthread
 
-FFTW_DIR = /usr/local/packages/fftw/3.2/intel-11.1-mvapich2-1.4
+FFTW3_DIR = /usr/local/packages/fftw/3.2/intel-11.1-mvapich2-1.4
 
 GIT_DIR = /usr/local/packages/git/1.6.1.3/intel-11.1
 
diff --git a/mdb/optionlists/supermuc.cfg b/mdb/optionlists/supermuc.cfg
index d8bbb3d..96be327 100644
--- a/mdb/optionlists/supermuc.cfg
+++ b/mdb/optionlists/supermuc.cfg
@@ -96,7 +96,7 @@ BLAS_DIR  = /lrz/sys/intel/mkl91/lib/em64t
 #BLAS_LIBS = mkl_lapack mkl_ipf guide pthread
 BLAS_LIBS = mkl_lapack guide pthread
 
-FFTW_DIR = /lrz/sys/libraries/fftw/3.3
+FFTW3_DIR = /lrz/sys/libraries/fftw/3.3
 
 GSL_DIR = /lrz/sys/libraries/gsl/1.15
 
diff --git a/mdb/optionlists/tezpur-mvapich2.cfg b/mdb/optionlists/tezpur-mvapich2.cfg
index f0ae9ba..8432ce5 100644
--- a/mdb/optionlists/tezpur-mvapich2.cfg
+++ b/mdb/optionlists/tezpur-mvapich2.cfg
@@ -84,7 +84,7 @@ F90_WARN_FLAGS =
 BLAS_DIR  = /usr/local/compilers/Intel/mkl-10.0/lib/em64t
 BLAS_LIBS = -Bstatic mkl_em64t guide -Bdynamic   pthread
 
-FFTW_DIR = /usr/local/packages/fftw-3.1.2-intel10.1
+FFTW3_DIR = /usr/local/packages/fftw-3.1.2-intel10.1
 
 GSL_DIR = /usr/local/packages/gsl-1.9-intel9.1
 
-- 
1.7.7.4

